aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1701,6 +1749,41 @@ impl VMAdapter for AptosVM {
             },
         })
     }
+
+    fn execute_single_transaction_sequential(
+        &self,
+        txn: &PreprocessedTransaction,
+        view: &impl StateView,
```

Comment:
You can make this `resolver: &impl MoveResolverExt` like in `execute_single_transaction` to avoid `&view.as_move_resolver()` calls?

Reply:
I initially had `resolver` as input instead of `view`. But in the second line of the function, I needed `view` when computing `vm_output.try_materialize(view)`.

Reply:
Maybe we can change materialization functions to use resolver instead of view as well ... No strong preference

Reply:
Will have to change the function signature of a series of functions and all their test cases along with it. I can do it if it's really that important.

Reply:
It's fine as is :)